api/scpca_portal/test/management/commands/test_create_portal_metadata.py

```diff
@@ -0,0 +1,48 @@
+import shutil
+
+from django.test import TransactionTestCase
+
+from scpca_portal import common
+from scpca_portal.management.commands import create_portal_metadata, load_data
+from scpca_portal.models import Library, Project, Sample
+
+# NOTE: Test data bucket is defined in `scpca_porta/common.py`.
+# When common.INPUT_BUCKET_NAME is changed, please delete the contents of
+# api/test_data/input before testing to ensure test files are updated correctly.
+
+ALLOWED_SUBMITTERS = {"scpca"}
+
+
+class TestCreatePortalMetadata(TransactionTestCase):
+    def setUp(self):
+        self.processor = create_portal_metadata.Command()
+        self.loader = load_data.Command()
+
+    @classmethod
+    def tearDownClass(cls):
+        super().tearDownClass()
+        shutil.rmtree(common.OUTPUT_DATA_PATH, ignore_errors=True)
+
+    def load_test_data(self):
+        # Expected object counts
+        PROJECTS_COUNT = 3
+        SAMPLES_COUNT = 9
+        LIBRARIES_COUNT = 7
+
+        self.loader.load_data(
+            allowed_submitters=list(ALLOWED_SUBMITTERS),
+            clean_up_input_data=False,
+            clean_up_output_data=False,
```

Comment:
Just a note: for this test you could have `cleanup_output_data=True` since you aren't testing any output of this command. This is purely just a shortcut to populate the data. This would give you a cleaner output `test_data/output` directory.

Reply:
Thank you for the note, David!